js/custom-menu.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @file
+ * Bootstrap dropdown on main menu.
+ */
+
+(function ($) {
+  Drupal.behaviors.exampleModule = {
+    attach: function (context, settings) {
+
+      $('.cd-dropdown .expanded a').on('click', function (e) {
+        $(this).parent().toggleClass('open');
+        e.stopPropagation();
+        e.preventDefault();
+      });
+
+    }
+  };
+}(jQuery));
```

Comment:
This works as-is, but it's helpful to future-us to declare the `Drupal` global just so we can keep track of what's going on. It makes linting go more smoothly too.

```js
(function ($, Drupal) {
  Drupal.behaviors.customMenu = {
    attach: function (context, settings) {
      // stuff
    }
  }
})(jQuery, Drupal);
```